overlord/devicestate/handlers.go

```diff
@@ -64,12 +97,7 @@ func (m *DeviceManager) doMarkPreseeded(t *state.Task, _ *tomb.Tomb) error {
 			// unmount all snaps
 			// TODO: move to snapstate.UnmountAllSnaps.
```

Comment:
Done in 6a8af413060b7e3b5fa0a64c4c1ebe0124acefb8. Since this is now in a more general-use place, I changed the implementation to truly unmount all snaps, rather than just the current ones. That was good enough for preseeding, but isn't actually "all" snaps.